packages/functions/src/jobs/SendConfirmation.test.ts

```diff
@@ -0,0 +1,132 @@
+import * as admin from "firebase-admin";
+import * as Functions from "../index";
+import {
+  AppointmentStatus,
+  BookingChange,
+  FunctionsApi,
+  Hospital,
+} from "@zm-blood-components/common/src";
+import { DbAppointment, DbDonor } from "../function-types";
+import {
+  deleteAppointmentsByIds,
+  getAppointmentsByIds,
+  setAppointment,
+} from "../dal/AppointmentDataAccessLayer";
+import { deleteDonor, setDonor } from "../dal/DonorDataAccessLayer";
+import { sampleUser } from "../testUtils/TestSamples";
+import { mocked } from "ts-jest/utils";
+import {
+  NotificationToDonor,
+  sendEmailToDonor,
+} from "../notifications/NotificationSender";
+import { SendConfirmationReminders } from "./CronJobs";
+import firebaseFunctionsTest from "../testUtils/FirebaseTestUtils";
+
+const wrapped = firebaseFunctionsTest.wrap(
+  Functions[FunctionsApi.BookAppointmentFunctionName]
+);
+
+jest.mock("../notifications/NotificationSender");
+const mockedNotifier = mocked(sendEmailToDonor);
+
+const DONORS = ["BookedDonorId", "AvailableDonorId", "OutOfRangeDonorId"];
+const APPOINTMENTS = [
+  "BookedAppointment",
+  "AvailableAppointment",
+  "OutOfRangeAppointment",
+];
```

Comment:
Add to the names something about the test name - this way we would know in the DB they belong to this test